mpd/asyncio.py

```diff
@@ -218,6 +218,10 @@ async def connect(self, host, port=6600, loop=None):
 
         self.__run_task = asyncio.Task(self.__run())
 
+    @property
+    def connected(self):
```

Comment:
@mweinelt please review.

Reply:
Makes sense, since we reset `self.__run_task` in https://github.com/Mic92/python-mpd2/pull/176/files#diff-a25a2d57c2fffef577da09ba28e87d060366e5dbc6056fa4c789ce7b689b8b1cR316 when we run into any Exception.